app/controllers/green_flag/admin/features_controller.rb

```diff
@@ -1,16 +1,27 @@
 class GreenFlag::Admin::FeaturesController < ApplicationController
-
   layout 'green_flag/application'
+  helper_method :flash_class
+
+  before_filter :find_feature, only: [:show, :destroy]
```

Comment:
I know it makes the code a little DRY, but I don't like this pattern.  I think it's clearer to have the instance variables set in the action method, instead of appearing from a filter.